src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -475,6 +468,46 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
     return '0'
   }, [opportunityType, virtualRuneDepositAmountFiatUserCurrency])
 
+  const thorchainNotationPoolAssetId = useMemo(() => {
+    if (!poolAsset) return undefined
+    return assetIdToPoolAssetId({
+      assetId: poolAsset.assetId,
+    })
+  }, [poolAsset])
+
+  const memo = useMemo(() => {
```

Comment:
Should we rename this to `feeEstimationMemo` for clarity?

Reply:
Actually why not! Nice addition to the comment 2bfce049bdf5ec8e1d63f022de8797b9002b5049